cristin-project/src/main/java/no/unit/nva/cristin/projects/model/cristin/adapter/CristinPersonToPerson.java

```diff
@@ -19,15 +20,20 @@ public Person apply(CristinPerson cristinPerson) {
 
     private Person convert(CristinPerson cristinPerson) {
         return new Person(
-            convertId(cristinPerson.cristinPersonId()),
+            convertId(cristinPerson.identifiedCristinPerson(), cristinPerson.cristinPersonId()),
             cristinPerson.firstName(),
             cristinPerson.surname(),
             cristinPerson.email(),
             cristinPerson.phone());
     }
 
-    private URI convertId(String cristinPersonId) {
-        return getNvaApiId(cristinPersonId, PERSON_PATH_NVA);
+    private URI convertId(Boolean identified, String cristinPersonId) {
```

Comment:
Nitpick / non-blocking:
I needed some time and your comment below to understand what this does and why.
Can you think if a better structure and/or method name here, to make it more obvious?

The only suggestion I can think of is this, but I'm not sure it's actually better:
```
private URI getPersonUri(Boolean identified, String cristinPersonId) {
    if (isNull(identified) || identified) {
        return getNvaApiId(cristinPersonId, PERSON_PATH_NVA);
    }
    return null;
}
```

Reply:
Updated PR